case-lib/pipeline.sh

```diff
@@ -20,7 +20,7 @@ func_pipeline_export()
         opt="-f"
         while [ $# -ne 0 ]
         do
-            opt=$opt" $1"
+            opt=$opt" '$1'"
```

Comment:
I don't know this code yet, could you give an example showing why this is required?

Reply:
[cmd_line_log.txt](https://github.com/thesofproject/sof-test/files/4479510/cmd_line_log.txt)
@marc-hb Take a look at the log attached, You should see why I add a `' '`